client/styles/main.styl

```diff
@@ -143,3 +156,17 @@ nav
 
 .month-users
   margin-bottom 20px
+
+@-moz-keyframes spin { 100% {
+    -moz-transform: rotate(360deg)
+  }
+}
+@-webkit-keyframes spin { 100% {
+    -webkit-transform: rotate(360deg)
+  }
+}
+@keyframes spin { 100% {
+    -webkit-transform: rotate(360deg)
+    transform:rotate(360deg)
+  }
+}
```

Comment:
You can remove most of these curly brackets as in https://github.com/ecohealthalliance/tater/blob/35ea7d15ed7ea8f231a712309d9881a5a52af138/packages/styles/mixins.import.styl#L138-L142